azkaban-db/src/main/java/azkaban/db/DatabaseOperator.java

```diff
@@ -117,16 +118,35 @@ public DatabaseOperator(final QueryRunner queryRunner) {
    * @return The number of rows updated.
    */
   public int update(final String updateClause, final Object... params) throws SQLException {
-    try {
-      return this.queryRunner.update(updateClause, params);
-    } catch (final SQLException ex) {
-      // todo kunkun-tang: Retry logics should be implemented here.
-      logger.error("update failed", ex);
-      if (this.dbMetrics != null) {
-        this.dbMetrics.markDBFailUpdate();
+    int retry = 0;
```

Comment:
Consider renaming to retryCount or retries, I prefer first one.

Reply:
will rename the variable.